universal/ubi8/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # syntax=docker/dockerfile:1.3-labs
 
-FROM quay.io/devfile/base-developer-image:ubi8-latest
+# https://quay.io/devfile/base-developer-image
```

Comment:
This doesn't work ...

```
$➔ ~/p/updateBaseImages.sh -b main -w `pwd` -f Dockerfile      -maxdepth 3 --no-commit --no-push 

# Checking /home/nboldt/0/developer-images/universal/ubi8/Dockerfile ...
error: Your local changes to the following files would be overwritten by checkout:
	base/ubi8/Dockerfile
	universal/ubi8/Dockerfile
Please commit your changes or stash them before you switch branches.
Aborting
error: cannot pull with rebase: You have unstaged changes.
error: please commit or stash them.
+ devfile/base-developer-image:???
- quay.io/devfile/base-developer-image:ubi8-latest
# Error reading registry from: /tmp/getLatestImageTags.sh -b devspaces-3-rhel-8  --quay -c devfile/base-developer-image -x latest|-source|next|nightly --tag ^ubi8-
```

Need to add support for querying a registry that has no :latest image, or else start publishing a :latest image to  https://quay.io/repository/devfile/base-developer-image?tab=tags&tag=latest (which might be cleaner)